src/main/java/core/basesyntax/HelloWorld.java

```diff
@@ -4,5 +4,20 @@
  * Feel free to remove this class and create your own.
  */
 public class HelloWorld {
+    public static void main(String[] args) {
+        Figure[] array = new Figure[6];
+        FigureSupplier figureSupplier = new FigureSupplier();
 
+        for (int i = 0; i <= 2; i++) {
```

Comment:
The loop condition `i <= 2` should be `i < 3` to avoid confusion and ensure consistency with the typical loop pattern. Both conditions are functionally equivalent here, but using `i < 3` is more conventional.